integrations/repo_branch_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integrations
+
+import (
+	"net/http"
+	"net/url"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/Unknwon/i18n"
+	"github.com/stretchr/testify/assert"
+)
+
+func testCreateBranch(t *testing.T, session *TestSession, user, repo, oldRefName, newBranchName string, expectedStatus int) string {
+	req := NewRequest(t, "GET", path.Join(user, repo, "branches/_new", oldRefName)+"?name="+url.QueryEscape(newBranchName))
+	resp := session.MakeRequest(t, req, expectedStatus)
+	assert.EqualValues(t, expectedStatus, resp.HeaderCode)
```

Comment:
Will fix, I did write this code before MakeRequest was changed and forgot to remove that line